docs/reference/style-guide.mdx

```diff
@@ -0,0 +1,262 @@
+
+```
+:::warning Tip title
+
+This is a tip.
+
+:::
+```
+
+:::note Markdown admonitions are sensitive to newlines
+
+Make sure you put a newline after the opening `:::` and before the closing `:::`.
+
+:::
+
+### Code blocks
+
+Surround code with three backticks to create a code block. Include the language after the first set of backticks to enable syntax highlighting.
+
+```bash
+echo "Hello, World!"
+```
+
+### Tables
+
+```
+| Header 1 | Header 2 |
+| -------- | -------- |
+| Cell 1   | Cell 2   |
+```
+| Header 1 | Header 2 |
+| -------- | -------- |
+| Cell 1   | Cell 2   |
+
+### Blockquotes
+
+```
+> This is a blockquote.
+```
+> This is a blockquote.
+
+### Horizontal rules
+
+```
+---
+```
+---
+
+### `<details />`
+
+<details>
+  <summary>Click to expand</summary>
+
+  This is the content of the details.
+</details>
+
+## Components
+
+### Admonition
+
+<Admonition type="tip">
+  This is a tip.
+</Admonition>
+
+### ActionCard and PrimaryCTA
+
+<ActionCard
+  title="Get Started"
+  description="Learn how to get started with the Cloud Posse Reference Architecture"
+  action={{
+    label: 'Get Started',
+    href: '/reference/style-guide'
+  }}>
+  <PrimaryCTA to="/reference/style-guide">See our style guide</PrimaryCTA>
+</ActionCard>
+
+### AsciinemaWidget
+
+<AsciinemaWidget src="https://asciinema.org/a/113643.json" />
+
+### AtmosWorkflow
+
+<AtmosWorkflow />
+
+### CardGroup
+
+<CardGroup>
+  <Card title="Card 1" description="Description 1" />
+  <Card title="Card 2" description="Description 2" />
+  <Card title="Card 3" description="Description 3" />
+</CardGroup>
+
+### CategoryList
+
+<CategoryList path="/layers/spacelift/tutorials/"/>
+
+### CollapsibleText
+
+<CollapsibleText title="This is a collapsible text">
+  This is the content of the collapsible text.
+</CollapsibleText>
+
+### Definition
+
+<Definition term="Term" description="Description of the term" />
+
+### DocCardList
+
+<DocCardList items={[
+  {
+    type: 'link',
+    label: 'Terraform Components',
+    description: 'Reusable Terraform "Root" Modules for AWS',
+    href: '/components'
+  },
+  {
+    type: 'link',
+    label: 'Terraform Modules',
+    description: 'Reusable Terraform "Child" Modules',
+    href: '/modules'
+  }
+]}
+/>
+
+### DismissibleDialog
+
+<DismissibleDialog title="This is a dismissible dialog">
+  This is the content of the dialog.
+</DismissibleDialog>
+
+### Intro
+
+<Intro>
+  This is the content of the intro.
+</Intro>
+
+### KeyPoints
+
+<KeyPoints title="Example keypoints...">
+- 'Reusable Terraform "Root" Modules for AWS'
+- 'Reusable Terraform "Child" Modules'
+</KeyPoints>
+
+### Note
+
+<Note>
+  This is the content of the note.
+</Note>
+
+### PillBox
+
+<PillBox items={['Pill 1', 'Pill 2', 'Pill 3']} />
+
+### Slider
+
+<Slider>
+  <Card title="Card 1" description="Description 1" />
+  <Card title="Card 2" description="Description 2" />
+  <Card title="Card 3" description="Description 3" />
+</Slider>
```

Comment:
![CleanShot 2024-08-20 at 09 19 29](https://github.com/user-attachments/assets/ebe1b689-4353-41c1-b1cf-a5ea40b19e02)
Doesn't seem to render right

Reply:
The sliders wasn't designed to work on a page with other content

Reply:
looking into this now

Reply:
sliders resolved with just code sample and link